lib/resque/scheduler.rb

```diff
@@ -8,6 +8,8 @@
 require_relative 'scheduler/signal_handling'
 require_relative 'scheduler/failure_handler'
 
+require 'resque/scheduler/railtie' if defined?(Rails)
```

Comment:
Based on the issue that came up in `resque` I think we need to do

```suggestion
require 'resque/scheduler/railtie' if defined?(Rails::Railtie)
```

Right?